frontend/packages/console-dynamic-plugin-sdk/tsconfig-base.json

```diff
@@ -11,7 +11,7 @@
     "experimentalDecorators": true,
     "noUnusedLocals": true,
     "skipLibCheck": true,
-    "lib": ["es2016", "es2017.object", "es2020.promise", "es2020.string", "dom"],
+    "lib": ["es2016", "es2017.object", "es2020.promise", "es2020.string", "dom", "dom.iterable"],
```

Comment:
> Could you explain this change?

@TheRealJon Added the `dom.iterable` because I got a TypeScript error that says "Type must have a '[Symbol.iterator]()' method that returns an iterator". I used the  spread operator  in`[...response.headers]` and currently it does not support asynchronous iterators. However I can use `Headers.forEach` instead but spread operator looks cleaner.

Reply:
What about `headers.entries().reduce(...)`?

Reply:
I suggested we remove that logic anyway, so I guess this doesn't matter much if we go that route.